packages/react-keytips/src/hooks/useHotkeys.ts

```diff
@@ -54,7 +55,11 @@ const isKeyMatchingKeyboardEvent = (
   return false;
 };
 
-export const useHotkeys = (hotkeys: Hotkey[], target?: Document) => {
+export const useHotkeys = (
+  hotkeys: Hotkey[],
+  invokeEvent: KeytipsProps['invokeEvent'] = 'keydown',
```

Comment:
nit: Consider creating a dedicated type `InvokeEvent = 'keydown' | 'keyup'` that could be used both as a parameter for a hook and as a prop type, rather than directly extracting it from the props.